libcodechecker/analyze/analyzers/analyzer_clang_tidy.py

```diff
@@ -116,3 +117,41 @@ def construct_analyzer_cmd(self, res_handler):
         except Exception as ex:
             LOG.error(ex)
             return []
+
+    @classmethod
+    def resolve_missing_binary(cls, configured_binary, env):
```

Comment:
This method is very similar for both analyzers. Maybe a good candidate to abstract out into a base class or utility function? There is no magic function in the python standard library to do this?